sign_language_datasets/datasets/dgs_corpus/dgs_corpus.py

```diff
@@ -130,6 +141,25 @@ def load_split(split_name: str) -> Dict[str, List[str]]:
     return split
 
 
+DEFAULT_FPS = 50
+
+
+class DgsCorpusConfig(SignDatasetConfig):
+    def __init__(self, data_type: Literal["document", "sentence"] = "document", split: str = None, **kwargs):
```

Comment:
I don't think we should use a different class of config just for this corpus, because that could be very confusing. If other users are like me, they first try to load the corpus with `SignDatasetConfig` and then get:

```python
AttributeError: Failed to construct dataset dgs_corpus: 'SignDatasetConfig' object has no attribute 'data_type'
```

I would opt for always using the same config class, with optional arguments such as `data_type`

Reply:
I'm quite against it... if we can raise a different error perhaps it would be better, but i don't wanna make the main config messy - it would be bad design, and future datasets will follow